zubhub_frontend/zubhub/src/views/activities/activities.jsx

```diff
@@ -15,45 +14,74 @@ import styles from '../../assets/js/styles/views/activities/activitiesStyles';
 import { makeStyles } from '@mui/styles';
 import ErrorPage from '../error/ErrorPage';
 import DefaultStyles from '../../assets/js/styles';
-import { Grid, Box, Typography } from '@mui/material';
+import { Grid, Tab, Tabs, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
 import LoadingPage from '../loading/LoadingPage';
 const useStyles = makeStyles(styles);
 
 function Activities(props) {
-  const location = useLocation();
   const classes = useStyles();
   const [loading, setLoading] = useState(true);
   const { activities } = useSelector(state => state);
-  let [activityList, setActivityList] = useState([]);
-
+  let [activityList, setActivityList] = useState({
+    published: [],
+    unPublishedActivities: [],
+    userActivities: []
+  });
+  const [tab, setTab] = useState("published");
   const commonClasses = makeStyles(DefaultStyles)();
+  const { t } = useTranslation();
```

Comment:
@brrkrmn can you check if we have `t` as part of the `props` passed to `Activities`? If we don't then something Is wrong and we should fix that (but not on this PR probably). However if `t` is in `props` then we should use it and not bother importing it again

Reply:
we have it in the props, changing the file now

Reply:
I think there are other PRs where you made translation changes to. For can check for this in those too